doc/lua_api.md

```diff
@@ -10411,6 +10411,9 @@ performance and computing power the practical limit is much lower.
     -- distribution of the biomes.
     -- Heat and humidity have average values of 50, vary mostly between
     -- 0 and 100 but can exceed these values.
+
+    weight = 1.0,
+    -- Relative weight of the biome in the Voronoi diagram
```

Comment:
It would probably be best to clarify that `weight > 0` and to link to something which properly defines weighted Voronoi diagrams, or to briefly explain it here.